iris-mpc-gpu/src/server/actor.rs

```diff
@@ -510,55 +509,55 @@ impl ServerActor {
         tracing::info!("Server Actor finished due to all job queues being closed");
     }
 
-    pub fn load_full_db(
-        &mut self,
-        left: &IrisCodeDbSlice,
-        right: &IrisCodeDbSlice,
-        db_size: usize,
-    ) {
-        assert!(
-            [left.0.len(), right.0.len(),]
-                .iter()
-                .all(|&x| x == db_size * IRIS_CODE_LENGTH),
-            "Internal DB mismatch, left and right iris code db sizes differ, expected {}, left \
-             has {}, while right has {}",
-            db_size * IRIS_CODE_LENGTH,
-            left.0.len(),
-            right.0.len()
-        );
-
-        assert!(
-            [left.1.len(), right.1.len()]
-                .iter()
-                .all(|&x| x == db_size * MASK_CODE_LENGTH),
-            "Internal DB mismatch, left and right mask code db sizes differ, expected {}, left \
-             has {}, while right has {}",
-            db_size * MASK_CODE_LENGTH,
-            left.1.len(),
-            right.1.len()
-        );
-
-        let db_lens1 = self
-            .codes_engine
-            .load_full_db(&mut self.left_code_db_slices, left.0);
-        let db_lens2 = self
-            .masks_engine
-            .load_full_db(&mut self.left_mask_db_slices, left.1);
-        let db_lens3 = self
-            .codes_engine
-            .load_full_db(&mut self.right_code_db_slices, right.0);
-        let db_lens4 = self
-            .masks_engine
-            .load_full_db(&mut self.right_mask_db_slices, right.1);
-
-        assert_eq!(db_lens1, db_lens2);
-        assert_eq!(db_lens1, db_lens3);
-        assert_eq!(db_lens1, db_lens4);
-
-        self.current_db_sizes = db_lens1;
-    }
-
-    pub fn register_host_memory(&self) {
+    // pub fn load_full_db(
```

Comment:
Can be removed I assume as it was used only in tests?

Reply:
yeah thx I forgot to delete this